tools/rpkg/NAMESPACE

```diff
@@ -2,12 +2,7 @@
 
 export(duckdb)
 export(duckdb_shutdown)
-export(duckdb_register)
```

Comment:
Is this intended?

Reply:
Done automatically by `devtools::document()`. We'll see along the way why this happened/what was missing when focusing more on documentation.

Reply:
@hannesmuehleisen Did you mean to export `duckdb_register()` and friends?

Reply:
Yes.